package.json

```diff
@@ -39,6 +39,7 @@
     "build": "npm run build-css && react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
+<<<<<<< HEAD
```

Comment:
You need to resolve this merge conflict.

The app build script reads the proxy configuration from this file, and the file is currently impossible to parse, so this is probably the reason your API requests are failing.